### DISCUSSION TITLE
无法推送PT种子文件，求解（报错信息undefined）

### DISCUSSION THREAD

**Discussion by zyw28** [ronggang/PT-Plugin-Plus#903] [Q&A] [answered]
使用黑群晖进行下载，设置完成了站点、下载目录，但一推送PT种子文件注提示下图：
![image](https://user-images.githubusercontent.com/56476413/133914726-9cff44b2-150e-4389-95bf-e2ef599ba07c.png)

后台查看有报错信息：
![image](https://user-images.githubusercontent.com/56476413/133914772-4875105b-6541-426e-af58-aad6ab1c1682.png)

![image](https://user-images.githubusercontent.com/56476413/133914788-ca5a2eed-ab12-4a17-82a9-c1b7440f3f08.png)

**Comment by leonmu** [comment]
情况相同 ，求解

**Comment by zyw28** [comment, marked as answer]
把群晖的下载Download Station，降级为老版本解决问题。不要升级，新版本定义和接口发生变化了，pt-plugin没有适配



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年12月21日(周二) 晚上9:53
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ronggang/PT-Plugin-Plus] 无法推送PT种子文件，求解（报错信息undefined） (Discussion #903)




 
情况相同 ，求解
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by leonmu** [reply]
降级到3.7.3版本管用
https://archive.synology.cn/download
安装过程中不选择删除资料库， 可以保留下载任务
感谢

**Comment by zyw28** [comment]
是的，但也请尽快匹配新版




醪糟粉子蛋
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ronggang/PT-Plugin-Plus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月22日(星期三) 上午10:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ronggang/PT-Plugin-Plus] 无法推送PT种子文件，求解（报错信息undefined） (Discussion #903)





 
降级到3.7.3版本管用
 https://archive.synology.cn/download
 安装过程中不选择删除资料库， 可以保留下载任务
 感谢
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
